test/e2e/infra/vagrant/push_antrea.sh

```diff
@@ -181,8 +200,30 @@ if [ "$FLOW_AGGREGATOR" == "true" ]; then
             ssh -F ssh-config k8s-node-control-plane kubectl apply -f elk-flow-collector/elk-flow-collector.yml -n elk-flow-collector
             LOGSTASH_CLUSTER_IP=$(ssh -F ssh-config k8s-node-control-plane kubectl get -n elk-flow-collector svc logstash -o jsonpath='{.spec.clusterIP}')
             ELK_ADDR="${LOGSTASH_CLUSTER_IP}:4739:udp"
-
             $THIS_DIR/../../../../hack/generate-manifest-flow-aggregator.sh --mode dev -fc $ELK_ADDR > "${FLOW_AGG_YML}"
+
+        elif [[ $FLOW_COLLECTOR == "Grafana" ]]; then
+            echo "Deploy ClickHouse flow collector"
+            # Generate manifest
+            $THIS_DIR/../../../../hack/generate-manifest-flow-visibility.sh --mode dev > "${FLOW_VIS_YML}"
+            $THIS_DIR/../../../../hack/generate-manifest-flow-aggregator.sh --mode dev -ch > "${FLOW_AGG_YML}"
+
+            # Push ClickHouse Monitor image
+            SAVED_CH_MONITOR_IMG=/tmp/flow-visibility-clickhouse-monitor.tar
+            CH_MONITOR_IMG_NAME=projects.registry.vmware.com/antrea/flow-visibility-clickhouse-monitor:latest
+            pushImgToNodes "$CH_MONITOR_IMG_NAME" "$SAVED_CH_MONITOR_IMG"
+
+            # Push images to nodes
```

Comment:
nit: this actually seems to copy manifests, rather than pushing images.

Reply:
You're right, I've modified it. Thanks.